From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Thu, 22 Aug 2019 04:53:01 +0300
Subject: meson: Don't install libtracker-common static library

As per GNOME/tracker!106 libtracker-common is now a static library, as per this
there's not point of installing it.

Origin: https://gitlab.gnome.org/GNOME/tracker/merge_requests/119
Applied-Upstream: 2.99.91
---
 src/libtracker-common/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/libtracker-common/meson.build b/src/libtracker-common/meson.build
index 4accbb1..041b7f0 100644
--- a/src/libtracker-common/meson.build
+++ b/src/libtracker-common/meson.build
@@ -44,8 +44,6 @@ libtracker_common = static_library('tracker-common',
   dependencies: tracker_common_dependencies + [unicode_library],
   c_args: tracker_c_args,
   include_directories: [configinc, srcinc],
-  install: true,
-  install_dir: tracker_internal_libs_dir,
 )
 
 commoninc = include_directories('.')
